addons/pkg/util/clusterbootstrapclone/clusterbootstrapclone.go

```diff
@@ -405,15 +425,42 @@ func (h *Helper) HandleExistingClusterBootstrap(clusterBootstrap *runtanzuv1alph
 		return nil, err
 	}
 
-	// ensure ownerRef of clusterBootstrap on created secrets and providers, this can only be done after
-	// clusterBootstrap is created
+	providers = append(providers, providerRefs...)
```

Comment:
Wouldnt appending add duplicates ? Shouldnt we be using providerRefs instead ?

Reply:
yes! you are right... at this point provider is a subset of providerRef

Reply:
but I guess we don't care about providers with valuesfrom.inline, so I think you are correct we can get away with just providerRef

Reply:
recreated list of providers with latest information from clusterbootstrap